### PR TITLE
(Fixes #436) PgpActivity should expose withInput, withOutput and markSuccessful

### LINKED ISSUE #436: PgpActivity should expose withInput, withOutput and markSuccessful

### PR DESCRIPTION
**Pull request by sethyates**

### CODE REVIEW

File: core/src/main/scala/com/krux/hyperion/DataPipelineDef.scala [hunk 1]
```diff
@@ -1,21 +1,12 @@
...
-import com.krux.hyperion.workflow.WorkflowExpressionImplicits
+import com.amazonaws.services.datapipeline.model.{PipelineObject => AwsPipelineObject}
+import com.krux.hyperion.aws.{AdpJsonSerializer, AdpPipelineSerializer}
+import com.krux.hyperion.workflow.WorkflowExpression
 import org.json4s.JsonDSL._
 import org.json4s.{JArray, JValue}
 
-import com.krux.hyperion.aws.{AdpJsonSerializer, AdpParameterSerializer, AdpPipelineSerializer}
-import com.krux.hyperion.workflow.WorkflowExpression
+import scala.language.implicitConversions
```
Comment (realstraw) [inline core/src/main/scala/com/krux/hyperion/DataPipelineDef.scala]: I think the import order should be
```
java.*
__blank line__
scala.*
__blank line__
com.other.*
__blank line__
com.krux.*
```

**Comment by realstraw** [comment]
👍

File: core/src/main/scala/com/krux/hyperion/DataPipelineDef.scala (hunk 1 above)
Comment (sethyates) [inline core/src/main/scala/com/krux/hyperion/DataPipelineDef.scala]: OK.  This is just whatever IntelliJ did to it.  I'll see if there's a setting.

Comment (sethyates) [inline core/src/main/scala/com/krux/hyperion/DataPipelineDef.scala]: Found it.  I'll re-apply

**Pull request merged**